packages/universe-components/src/ui/receipt-status.tsx

```diff
@@ -29,6 +29,7 @@ interface WrapperProps {
 }
 
 function ReceiptBodyBuilder(status: string): React.JSX.Element {
+  console.log('STATUS',status)
   switch (status) {
     case 'Claim':
```

Comment:
Can you make these upper-case as well?

Reply:
Why are they uppercase in the first place? Are they being used in title case anywhere else? That was a quick fix to make it work but we should decide the casing and stick to it